batch/batch/google_storage.py

```diff
@@ -34,19 +34,24 @@ def __init__(self, blocking_pool, *, project=None, credentials=None):
         self._wrapped_delete_gs_files = self._wrap_network_call(GCS._delete_gs_files)
 
     async def write_gs_file(self, uri, string, *args, **kwargs):
-        return await self._wrapped_write_gs_file(self, uri, string, *args, **kwargs)
+        return await retry_transient_errors(self._wrapped_write_gs_file,
```

Comment:
can you run a test where you write to the same uri the same string twice? I'm worried that we need to specify an overwrite command or something.

Reply:
It worked fine.